interfaces/builtin/network_control.go

```diff
@@ -92,6 +92,9 @@ network sna,
 /{,usr/}{,s}bin/wpa_passphrase ixr,
 /{,usr/}{,s}bin/wpa_supplicant ixr,
 
+/dev/rfkill rw,
+/dev/rfkill r,
```

Comment:
I tried with network-manager that has rw, and it still did not work. There are other interfaces with file both access r and rw